### DISCUSSION TITLE
Using Mistral-7B model

### DISCUSSION THREAD

**Discussion by pgp69** [PromtEngineer/localGPT#595] [Q&A] [answered]
how to use mistral-7b model, it returns a message that it is not supported ;(

This is the model: NickyNicky/Mistral-7B-OpenOrca-oasst_top1_2023-08-25-v2

**Comment by PromtEngineer** [comment, marked as answer]
Look for the model in gguf format. You should be able to use that.

**Reply by pgp69** [reply]
Can I convert a mistral model to GGUF. I don't success using RTX3050/4GB of RAM with cuda

I am running exactly the installation instructions for CMAKE_ARGS="-DLLAMA_CUBLAS=on" FORCE_CMAKE=1 pip install llama-cpp-python==0.1.83

but does not work  with both Win11 and WSL-2.

Torch.cuda.is_available() -> True

Does it make sense to use an eGPU with RTX-3060/12GB RAM over Thunderbolt-3. Will the performance  be very low?

**Comment by clearsitedesigns** [comment]
This is a great source of models updates frequently;

https://huggingface.co/TheBloke

**Reply by pgp69** [reply]
I tried this model works very good, I'm looking for a model to fine tuning  to work on Bulgarian, for now without much success
Thank You.
